### PR TITLE
Nextcloud support 

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: _posts/2021-07-04-nextcloud-support.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+site: freiburg
+tags: [tools]
+title: ownCloud and Nextcloud integration for your private data
+supporters:
+- denbi
+- elixir
+---
+
+[ownCloud](https://owncloud.com/) and [Nextcloud](https://nextcloud.com/) are free software solutions for the storage and management of data.
+Many universities are offering own Nextcloud instances where researchers can store and share data in a GDPR compliant way.
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:11]:
```suggestion
Many universities are offering their own Nextcloud instances where researchers can store and share data in a GDPR compliant way.
```

File: _posts/2021-07-04-nextcloud-support.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+We have now added support to Galaxy to interact with your storage. You can import data from you Nextcloud instance or export data from your history
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:13]:
```suggestion
We have now added support to Galaxy to interact with your storage. You can import data from your Nextcloud instance or export data from your history
```

File: _posts/2021-07-04-nextcloud-support.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+to your cloud storage. Technically this works by using the [WebDAV](https://en.wikipedia.org/wiki/WebDAV) protocol and you can use this integration with
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:14]:
```suggestion
to your cloud storage. Technically, this works by using the [WebDAV](https://en.wikipedia.org/wiki/WebDAV) protocol and you can use this integration with
```

File: _posts/2021-07-04-nextcloud-support.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+every server that speaks this protocol. In the following we will give you a few examples how to configure your Galaxy account to make use of that feature.
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:15]:
```suggestion
every server that speaks this protocol. 

Here are some examples of how to configure your Galaxy account to make use of that feature.
```

File: _posts/2021-07-04-nextcloud-support.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+![](/assets/media/b2drop_access.png)
+
+
+For example to get access to the [B2Drop service of EUDAT](https://eudat.eu/services/b2drop). You can configure your settings like:
+    Server Domain: https://b2drop.bsc.es
+    Server Path: /remote.php/webdav
+    Username: <your username> (e.g. your email)
+    Password: <your password>
+
+For the CRC1452 you ...
```
Comment (bgruening) [inline _posts/2021-07-04-nextcloud-support.md:28]: @pavanvidem could you please test this with the CRC1452 Nextcloud and add the example here?

File: _posts/2021-07-04-nextcloud-support.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+We have now added support to Galaxy to interact with your storage. You can import data from you Nextcloud instance or export data from your history
+to your cloud storage. Technically this works by using the [WebDAV](https://en.wikipedia.org/wiki/WebDAV) protocol and you can use this integration with
+every server that speaks this protocol. In the following we will give you a few examples how to configure your Galaxy account to make use of that feature.
+
+In all cases you need to go User → Preferences → [Manage Information](https://usegalaxy.eu/user/information)
+
+![](/assets/media/b2drop_access.png)
+
+
+For example to get access to the [B2Drop service of EUDAT](https://eudat.eu/services/b2drop). You can configure your settings like:
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:22]:
```suggestion
For example, to get access to the [B2DROP service of EUDAT](https://eudat.eu/services/b2drop), you can configure your settings like:
```

File: _posts/2021-07-04-nextcloud-support.md (hunk 5 above)
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:28]:
```suggestion
For the CRC1452, you...
```

**Comment by bgruening** [comment]
Could be added later if we figure out the auth.
```
For [bwSync&Share](https://www.rz.uni-freiburg.de/services/bwdienste/bwsyncshare) you can try the following settings:

>  Server Domain: `https://bwsyncandshare.kit.edu`
>
>  Server Path: `/remote.php/webdav`
>
>  Username: `<your username>` (e.g. your email)
>
>  Password: `<your password>`

```

File: _posts/2021-07-04-nextcloud-support.md (hunk 5 above)
Comment (bgruening) [reply _posts/2021-07-04-nextcloud-support.md]: I tested CRC1425 and it seems to work :) Something for your retreat next week :)

**Comment by bgruening** [comment]
Ready, if you like it merge it :)

File: _posts/2021-07-04-nextcloud-support.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+>  Server Domain: `https://nxc-1425.imbi.uni-freiburg.de`
+>
+>  Server Path: `/remote.php/webdav`
+>
+>  Username: `<your username>` (e.g. your email)
+>
+>  Password: `<your password>`
+
+And don't forget you can also export files back to your cloud storage.
```
Comment (beatrizserrano) [inline _posts/2021-07-04-nextcloud-support.md:57]:
```suggestion
And don't forget that you can also export files back to your cloud storage.
```

**Pull request merged**